src/navigation/RootStackNavigation.tsx

```diff
@@ -255,9 +259,7 @@ const AppUnlockHandler = ({ children }: { children: ReactNode }) => {
       const error = e as { message?: string }
 
       if (error.message?.includes('User canceled')) {
-        Alert.alert('Authentication required', 'Please authenticate to unlock your wallet.', [
-          { text: 'Try again', onPress: unlockApp }
-        ])
+        showToast('Authentication required. Exit the app and try again.')
```

Comment:
Showing an alert was creating this problem: https://github.com/alephium/mobile-wallet/issues/344

So I thought we should quit the app if the user cancels authentication. But after researching I found out that there's no cross-platform native way to do this and I also read comments that iOS prevents apps from closing themselves.

The simpler solution to this is to display a toast and ask the user to refresh the app.